cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
+title: Estimate Serverless Usage with MovR
+summary: Run the MovR sample application to estimate RU usage in CockroachDB Serverless.
+toc: true
+referral_id: docs_serverless_movr
+docs_area: get_started
+---
+
+This tutorial shows you how to initialize and run the MovR workload on a {{ site.data.products.serverless }} cluster, then use the Request Unit consumption data to estimate a sufficient budget for the MovR workload.
```

Comment:
First mention of 'movr' could link to the docs that describe what it is, describes schema etc: https://www.cockroachlabs.com/docs/v22.1/movr.html